app/assets/stylesheets/signup_form.css

```diff
@@ -0,0 +1,5 @@
+.field-hint {
```

Comment:
This style isn't necessary, we should reuse the `.help-block` style from `style.css.scss` to be consistent with other forms